assets/src/application/ui/display/icons/svgs/sort.svg

```diff
@@ -0,0 +1,11 @@
+<svg
+	aria-hidden="true"
+	focusable="false"
+	xmlns="http://www.w3.org/2000/svg"
+	version="1.1"
+	height="1.25rem"
+	viewBox="0 0 20 20"
+	width="1.25rem"
+>
+<path d="M11 7h-10l5 7zM9 14h10l-5-7z" />
+</svg>
```

Comment:
got it from dashicons, hope it's fine:

https://github.com/WordPress/dashicons/blob/master/sources/svg/sort.svg

Reply:
love it !!!